advanced/advanced_numpy/index.rst

```diff
@@ -124,10 +124,20 @@ Reminder: two :class:`ndarrays <ndarray>` may share the same memory::
 
 Memory does not need to be owned by an :class:`ndarray`::
 
-    >>> x = '1234'
+    >>> x = '1234'      # In Python 3, use:   x = b'1234'
+
+.. For Python3
+   >>> try:
```

Comment:
Why the try/except? ``x = b'1234'`` is valid Python 2 code, the 'b' is just ignored.